client/src/components/AuditAdmin/timers.test.tsx

```diff
@@ -52,7 +52,8 @@ const loadEach = [
   aaApiCalls.getSettings(auditSettings.all),
 ]
 
-describe('timers', () => {
+// TODO: Fix these tests after switching to Vite
```

Comment:
I got some weird errors with the fake timers. I think I will likely need to upgrade Jest, but can't do that until after ejecting react-scripts (which controls Jest version).